src/commands/status/status.ts

```diff
@@ -42,16 +42,8 @@ export const status = async (options: OptionValues, command: BaseCommand) => {
     Name: user.full_name,
     Email: user.email,
     GitHub: ghuser,
+    Teams: accounts.map(({ name }) => name),
```

Comment:
🤔 so if it's true that this is the only use of `netlify.{cachedConfig\.,}accounts`, should we do a fast-follow PR to remove it and query here? 😈 We should take any oppportunity to remove stuff from BaseCommand, I think.

Reply:
Probably. I'd think a little bigger: I'd like to figure out what the impact of lifting _all_ of the API calls out of `@netlify/config` would be so we have better control over startup sequence timing/reduce unnecessary calls/investigate a short-lived persistent cache.